src/main/java/edu/harvard/iq/dataverse/DatasetVersion.java

```diff
@@ -1772,270 +1763,17 @@ public String getPublicationDateAsString() {
         return r;
     }
 
-    // TODO: Consider moving this comment into the Exporter code.
-    // The export subsystem assumes there is only
-    // one metadata export in a given format per dataset (it uses the current 
-    // released (published) version. This JSON fragment is generated for a 
-    // specific released version - and we can have multiple released versions. 
-    // So something will need to be modified to accommodate this. -- L.A.  
     /**
      * We call the export format "Schema.org JSON-LD" and extensive Javadoc can
-     * be found in {@link SchemaDotOrgExporter}.
+     * be found in {@link edu.harvard.iq.dataverse.export.SchemaDotOrgExporter}.
      */
+    @Deprecated
```

Comment:
Based on my earlier comments  - if this gets used in producing the schema.org header, we still need it.